### PR TITLE
Squid plugins

### PR DESCRIPTION
**Pull request by evbevz**
Includes moved to right places.
Auth plugins include added.

### CODE REVIEW

File: src/opnsense/service/templates/OPNsense/Proxy/squid.conf [hunk 1]
```diff
@@ -258,10 +258,14 @@ icap_enable off
 {% include ['OPNsense/Proxy/squid.user.alt_auth.conf', 'OPNsense/Proxy/squid.user.local_auth.conf'] %}
 {% endif %}
 
+# Pre-auth plugins
+include /usr/local/etc/squid/pre-auth/*.conf
 {% include "OPNsense/Proxy/squid.user.pre_auth.conf" ignore missing with context %}
```
Comment (fichtner) [inline src/opnsense/service/templates/OPNsense/Proxy/squid.conf]: I guess this should be removed, @AdSchellevis  ?

File: src/opnsense/service/templates/OPNsense/Proxy/squid.conf [hunk 2]
```diff
@@ -258,10 +258,14 @@ icap_enable off
...
 
 {% include "OPNsense/Proxy/squid.acl.conf" ignore missing with context %}
 
+# Post-auth plugins
+include /usr/local/etc/squid/post-auth/*.conf
 {% include "OPNsense/Proxy/squid.user.post_auth.conf" ignore missing with context %}
```
Comment (fichtner) [inline src/opnsense/service/templates/OPNsense/Proxy/squid.conf]: this too

File: src/opnsense/service/templates/OPNsense/Proxy/squid.conf (hunk 1 above)
Comment (AdSchellevis) [inline src/opnsense/service/templates/OPNsense/Proxy/squid.conf]: @fichtner I will do this after inspection of the plugins

Comment (fichtner) [inline src/opnsense/service/templates/OPNsense/Proxy/squid.conf]: ok

**Pull request merged**